web-app/CKS.Web/Startup.cs

```diff
@@ -51,16 +47,40 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 
 			app.UseDefaultFiles();
-			app.UseStaticFiles();
+			app.UseStaticFiles(new StaticFileOptions
+			{
+				OnPrepareResponse = ctx =>
+				{					
+					var fileName = ctx.File.Name;
```

Comment:
Can you add a comment as to why we're using these headers i.e. explain they're for the gatsby generated files and add a link to the caching page on the gatsby docs explaining why - otherwise these values look a bit arbitrary/hard to understand why we've chosen these?